modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesRuntimeProvisioningService.java

```diff
@@ -184,17 +184,21 @@ public void archiveOrganization(TenantInfo tenantInfo) throws RuntimeProvisionin
             kubClient = KubernetesProvisioningUtils.getFabric8KubernetesClient();
             DeploymentList deploymentList = kubClient.extensions().deployments().list();
 
-            if (deploymentList.getItems().contains(deployment)) {
+            Deployment currentDeployement = kubClient.inNamespace(namespace.getMetadata().getName()).extensions()
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/c707570879201754759203ff99b9c5ea5489dcba